code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -38,7 +38,8 @@
 	if (A)
 		if(ismob(A))
 			toxmob(A)
-		if((istype(A,/obj/machinery/the_singularitygen))||(istype(A,/obj/singularity/)))
+		if((istype(A,/obj/machinery/the_singularitygen))||(istype(A,/obj/singularity/))||(istype(A,/obj/singularity/energy_ball
```

Comment:
isn't this excessive? energy_ball is a child of singularity, which is already checked for by istype.